Bottom: 4bc9717ae3945b6d9303a0dedbc923bf66c4021d
Top:    731444a98f6ce16245b94298956a861253e6d508
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 21:02:33 +0100

hid/gtk: DrawMark from ghid_invalidate_all() as well as DrawAttached


---

diff --git a/src/hid/gtk/gtkhid-gdk.c b/src/hid/gtk/gtkhid-gdk.c
index c9947c2..fa6dea0 100644
--- a/src/hid/gtk/gtkhid-gdk.c
+++ b/src/hid/gtk/gtkhid-gdk.c
@@ -772,6 +772,7 @@ ghid_invalidate_all ()
   ghid_draw_grid ();
 
   DrawAttached ();
+  DrawMark ();
   ghid_screen_update ();
 }
